.github/workflows/default.yml

```diff
@@ -13,7 +13,11 @@ jobs:
   test:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
+        with:
+          # Number of commits to fetch. 0 indicates all history for all branches and tags.
```

Comment:
why do we need this? I've never seen it before

Reply:
I'll add a comment as to why. One of the tests brought in for the brew rules depends on the workspace status command which depends on tags but the default checkout step uses a depth of 1 and doesn't fetch any tags which fails the test